src/NuGet.Indexing/Extraction/DocumentCreator.cs

```diff
@@ -261,6 +262,61 @@ private static void AddRequiresLicenseAcceptance(Document document, IDictionary<
             }
         }
 
+        private static void AddPackageTypes(Document document, IDictionary<string, string> package)
```

Comment:
Not sure we want to store JSON in the search index. Can we just store the raw values and have the API generate JSON if needed? All JSON formatting is additional cruft that will make the index grow in size.